### ISSUE TITLE
Installation issue mac

### ISSUE THREAD

**Issue by stefanknegt** [Blosc/bcolz#407]
When I try to install bcolz, either using pip or from source, I get one of the following errors:

`error: command 'gcc' failed with exit status 1`
`error: command 'xcrun' failed with exit status 1`

Often this is caused by not having Xcode or Xcode select installed, but I have both installed and have no issues with other packages. Does anyone have an idea how to fix this?

**Comment by stefanknegt** [comment]
Somehow this issue is not present when I install version 1.2.0 instead of 1.2.1.

**Comment by binary-signal** [comment]
have you tried to update pip setuptools and wheel before trying to install bcolz?
`python -m pip install pip setuptools wheel -U`

**Comment by stefanknegt** [comment]
Yes I've tried that, but no luck. I even reinstalled python and tried different versions of python, but nothing (except installing 1.2.0) seems to work.
